libcxx/include/variant

```diff
@@ -1273,6 +1293,22 @@ public:
     __impl_.__swap(__that.__impl_);
   }
 
+#  if _LIBCPP_STD_VER >= 26
+  // [variant.visit], visitation
+
+  template <int = 0, class _Self, class _Visitor>
```

Comment:
Would it be better to use a special tag type like this
```C++
struct __variant_visit_barrier_tag { // unnamable when using standard library modules
  explicit __variant_visit_barrier_tag() = default;
};
// ...
template <__variant_visit_barrier_tag = __variant_visit_barrier_tag{}, class _Self, class _Visitor>
// ...
```
to avoid accepting `v.visit<0, T, F>(f)`?